tests/test_agents.py

```diff
@@ -464,7 +464,7 @@ async def gen_answer(self, state) -> str:  # noqa: ARG001
 @pytest.mark.flaky(reruns=3, only_rerun=["AssertionError", "EmptyDocsError"])
 @pytest.mark.asyncio
 async def test_agent_sharing_state(
-    agent_test_settings: Settings, subtests: SubTests, callback_type: str
+    agent_test_settings: Settings, subtests: SubTests, callback_type: str | None
```

Comment:
added as this was an unresolved comment from James on https://github.com/Future-House/paper-qa/pull/590

Reply:
Thanks!